Readme: Add Terms of Service (#4932)
swissspidy/web-stories-wp: readme.txt

3. Explore our WYSIWYG editor with easy access to many editing tools (shown here: zooming and cropping media)
4. Get started building your first story by editing one of our expressive, fully built template stories included with the editor.

> == Terms of Service ==

Reviewer: This is actually not one of the default allowed headings. It needs to use `===` and moved right before `== Installation ==`, otherwise it won't be shown on the main page.

> By using this plugin, you agree to [Google's Terms of Service](https://policies.google.com/terms). By using third-party imagery and video provided by [Unsplash](https://unsplash.com/terms) and [Coverr](https://coverr.co/license), you agree to adhere to their ToS and licensing terms."

Reviewer: ```suggestion
By using this plugin, you agree to [Google's Terms of Service](https://policies.google.com/terms). By using third-party imagery and video provided by [Unsplash](https://unsplash.com/terms) and [Coverr](https://coverr.co/license), you agree to adhere to their ToS and licensing terms.
```

Errant apostrophe

Reviewer: By the way, should we also include Tenor (https://tenor.com/legal-terms) here?

> By using this plugin, you agree to [Google's Terms of Service](https://policies.google.com/terms). By using third-party imagery and video provided by [Unsplash](https://unsplash.com/terms) and [Coverr](https://coverr.co/license), you agree to adhere to their ToS and licensing terms."

Reviewer 2: ```suggestion
By using this plugin, you agree to [Google's Terms of Service](https://policies.google.com/terms). By using third-party imagery and video provided by [Unsplash](https://unsplash.com/terms), [Coverr](https://coverr.co/license) and [Tenor](https://tenor.com/legal-terms), you agree to adhere to the respective Terms of Service.
```

A suggestion to include @swissspidy 's comment. Also, spelling out ToS as Terms of Service as I don't think it's defined anywhere.
